test/functional/ui/inccommand_spec.lua

```diff
@@ -32,7 +32,7 @@ local multiline_text = [[
   7 8 9
 ]]
 
-local multimatch_text  = [[
```

Comment:
Don't run stylua on the test files.

Reply:
Done. Could you explain the reason why it's not run there by default?

Reply:
Because the screen tests need to be exactly like they are, or they will fail.